File: src/scanners/SteamFlatpakSource.js
const SteamSource = require("./SteamSource.js");
const SteamFlatpakGame = require("../games/SteamFlatpakGame.js");

const USER_DIR = process.env["HOME"];

class SteamFlatpakSource extends SteamSource{

	static name = "Steam (Flatpak)";
	static gameClass = SteamFlatpakGame;

	steamDir = `${USER_DIR}/.var/app/com.valvesoftware.Steam/.local/share/Steam`;

}

module.exports = SteamFlatpakSource;

File: gali/src/sources/steam/flatpak/steam_flatpak_source.py
from gali.sources.steam.steam_source import SteamSource
from gali.sources.steam.steam_xdg_game import SteamXDGGame
from gali.utils.locations import HOME


class SteamFlatpakSource(SteamSource):

    name: str = "Steam (Flatpak)"
    game_class: type[SteamXDGGame] = SteamXDGGame
    steam_dir: str = f"{HOME}/.var/app/com.valvesoftware.Steam/.local/share/Steam"
